### DISCUSSION TITLE
Github actions is not working fine getting the errors.

### DISCUSSION THREAD

**Discussion by ayanSimpplr** [community/community#69088] [Actions]
### Select Topic Area

General

### Body

Hi there,

We are encountering issues with the Kubernetes selfhosted Actions Runner controller. It stops in the middle while running the jobs. Could you please investigate? I am sharing the logs.

Github pod logs:
`[WORKER` 2023-10-03 04:18:30Z INFO ActionManifestManager] } [WORKER 2023-10-03 04:18:30Z INFO ActionManifestManager] Input 'github-server-url': default value evaluate result: "" [WORKER 2023-10-03 04:18:30Z INFO ExtensionManager] Getting extensions for interface: 'GitHub.Runner.Worker.IActionCommandExtension' [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command internal-set-repo-path [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command set-env [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command set-output [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command save-state [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command add-path [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command add-mask [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command add-matcher [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command remove-matcher [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command warning [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command error [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command notice [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command debug [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command group [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command endgroup [WORKER 2023-10-03 04:18:30Z INFO ActionCommandManager] Register action command extension for command echo [WORKER 2023-10-03 04:18:30Z INFO HostContext] Well known directory 'Bin': '/home/runner/bin' [WORKER 2023-10-03 04:18:30Z INFO HostContext] Well known directory 'Root': '/home/runner' [WORKER 2023-10-03 04:18:30Z INFO HostContext] Well known directory 'Externals': '/home/runner/externals' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Starting process: [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   File name: '/home/runner/externals/node16/bin/node' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   Arguments: '"/home/runner/_work/_actions/actions/checkout/v3/dist/index.js"' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   Working directory: '/home/runner/_work/Simpplr-Notification/Simpplr-Notification' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   Require exit code zero: 'False' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   Encoding web name:  ; code page: '' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   Force kill process on cancellation: 'False' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   Redirected STDIN: 'False' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   Persist current code page: 'False' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   Keep redirected STDIN open: 'False' [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]   High priority process: 'False' [WORKER 2023-10-03 04:18:30Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:30Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_4aae4f11-95da-52f2-ca84-e260f4a316db.1 [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Failed to update oom_score_adj for PID: 855. [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] System.UnauthorizedAccessException: Access to the path '/proc/855/oom_score_adj' is denied. [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]  ---> System.IO.IOException: Permission denied [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]    --- End of inner exception stack trace --- [WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper]    at System.IO.RandomAccess.WriteAtOffset(SafeFileHandle handle, ReadOnlySpan1 buffer, Int64 fileOffset)
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] at System.IO.Strategies.BufferedFileStreamStrategy.FlushWrite()
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] at System.IO.Strategies.BufferedFileStreamStrategy.Dispose(Boolean disposing)
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] at System.IO.StreamWriter.CloseStreamFromDispose(Boolean disposing)
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] at System.IO.StreamWriter.Dispose(Boolean disposing)
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] at System.IO.File.WriteAllText(String path, String contents)
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] at GitHub.Runner.Sdk.ProcessInvoker.WriteProcessOomScoreAdj(Int32 processId, Int32 oomScoreAdj)
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Sending SIGINT to process 855.
[WORKER 2023-10-03 04:18:30Z INFO JobServerQueue] Try to append 1 batches web console lines for record '267f69b3-604a-44ee-87d2-b53535db2d7c', success rate: 1/1.
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] STDOUT/STDERR stream read finished.
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] STDOUT/STDERR stream read finished.
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Successfully send SIGINT to process 855.
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Waiting for process exit or 7.5 seconds after SIGINT signal fired.
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Process exit successfully.
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Process cancelled successfully through Ctrl+C/SIGINT.
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Process started with process id 855, waiting for process exit.
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Process Cancellation finished.
[WORKER 2023-10-03 04:18:30Z INFO ProcessInvokerWrapper] Finished process 855 with exit code 130, and elapsed time 00:00:00.3387850.
[WORKER 2023-10-03 04:18:30Z INFO CreateStepSummaryCommand] Step Summary file (/home/runner/_work/_temp/_runner_file_commands/step_summary_1257eb98-b75e-477d-bd97-1d5f0ed2085a) is empty; skipping attachment upload
[WORKER 2023-10-03 04:18:30Z ERR StepsRunner] Caught cancellation exception from step: System.OperationCanceledException: The operation was canceled.
[WORKER 2023-10-03 04:18:30Z ERR StepsRunner] at System.Threading.CancellationToken.ThrowOperationCanceledException()
[WORKER 2023-10-03 04:18:30Z ERR StepsRunner] at GitHub.Runner.Sdk.ProcessInvoker.ExecuteAsync(String workingDirectory, String fileName, String arguments, IDictionary2 environment, Boolean requireExitCodeZero, Encoding outputEncoding, Boolean killProcessOnCancel, Channel1 redirectStandardIn, Boolean inheritConsoleHandler, Boolean keepStandardInOpen, Boolean highPriorityProcess, CancellationToken cancellationToken)
[WORKER 2023-10-03 04:18:30Z ERR StepsRunner] at GitHub.Runner.Common.ProcessInvokerWrapper.ExecuteAsync(String workingDirectory, String fileName, String arguments, IDictionary2 environment, Boolean requireExitCodeZero, Encoding outputEncoding, Boolean killProcessOnCancel, Channel1 redirectStandardIn, Boolean inheritConsoleHandler, Boolean keepStandardInOpen, Boolean highPriorityProcess, CancellationToken cancellationToken)
[WORKER 2023-10-03 04:18:30Z ERR StepsRunner] at GitHub.Runner.Worker.Handlers.DefaultStepHost.ExecuteAsync(IExecutionContext context, String workingDirectory, String fileName, String arguments, IDictionary2 environment, Boolean requireExitCodeZero, Encoding outputEncoding, Boolean killProcessOnCancel, Boolean inheritConsoleHandler, String standardInInput, CancellationToken cancellationToken) [WORKER 2023-10-03 04:18:30Z ERR  StepsRunner]    at GitHub.Runner.Worker.Handlers.NodeScriptActionHandler.RunAsync(ActionRunStage stage) [WORKER 2023-10-03 04:18:30Z ERR  StepsRunner]    at GitHub.Runner.Worker.ActionRunner.RunAsync() [WORKER 2023-10-03 04:18:30Z ERR  StepsRunner]    at GitHub.Runner.Worker.StepsRunner.RunStepAsync(IStep step, CancellationToken jobCancellationToken) [WORKER 2023-10-03 04:18:30Z INFO StepsRunner] Step result: Canceled [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext] Publish step telemetry for current step { [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "action": "actions/checkout", [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "ref": "v3", [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "type": "node16", [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "stage": "Post", [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "stepId": "267f69b3-604a-44ee-87d2-b53535db2d7c", [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "result": "canceled", [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "errorMessages": [ [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]     "The operation was canceled." [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   ], [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "executionTimeInSeconds": 2, [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "startTime": "2023-10-03T04:18:29.678742Z", [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext]   "finishTime": "2023-10-03T04:18:30.7065324Z" [WORKER 2023-10-03 04:18:30Z INFO ExecutionContext] }. [WORKER 2023-10-03 04:18:30Z INFO StepsRunner] No need for updating job result with current step result 'Canceled'. [WORKER 2023-10-03 04:18:30Z INFO StepsRunner] Current state: job state = 'Failed' [WORKER 2023-10-03 04:18:30Z INFO JobRunner] Finalize job. [WORKER 2023-10-03 04:18:30Z INFO JobServerQueue] Try to append 1 batches web console lines for record '267f69b3-604a-44ee-87d2-b53535db2d7c', success rate: 1/1. [WORKER 2023-10-03 04:18:30Z INFO HostContext] Well known directory 'Bin': '/home/runner/bin' [WORKER 2023-10-03 04:18:30Z INFO HostContext] Well known directory 'Root': '/home/runner' [WORKER 2023-10-03 04:18:30Z INFO HostContext] Well known directory 'Diag': '/home/runner/_diag' [WORKER 2023-10-03 04:18:30Z INFO HostContext] Well known directory 'Bin': '/home/runner/bin' [WORKER 2023-10-03 04:18:30Z INFO HostContext] Well known directory 'Root': '/home/runner' [WORKER 2023-10-03 04:18:30Z INFO HostContext] Well known directory 'Diag': '/home/runner/_diag' [WORKER 2023-10-03 04:18:30Z INFO JobExtension] Initialize Env context [WORKER 2023-10-03 04:18:30Z INFO JobExtension] Initialize steps context [WORKER 2023-10-03 04:18:30Z INFO JobExtension] Support log upload starting. [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Bin': '/home/runner/bin' [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Root': '/home/runner' [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Work': '/home/runner/_work' [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Temp': '/home/runner/_work/_temp' [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Bin': '/home/runner/bin' [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Root': '/home/runner' [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Diag': '/home/runner/_diag' [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Bin': '/home/runner/bin' [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Root': '/home/runner' [WORKER 2023-10-03 04:18:31Z INFO HostContext] Well known directory 'Diag': '/home/runner/_diag' [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Support log upload complete. [WORKER 2023-10-03 04:18:31Z INFO JobServerQueue] Try to append 1 batches web console lines for record '47ad5bda-ffab-4f20-9c50-5801efd60e32', success rate: 1/1. [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Total accessible running process: 30. [WORKER 2023-10-03 04:18:31Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:31Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_03a50df5-fe47-5dfa-f7f2-c176a9cbf361.1 [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 304 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/304/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 305 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/305/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 306 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/306/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 307 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/307/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 308 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/308/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 309 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/309/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 318 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/318/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 327 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/327/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 328 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/328/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 337 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/337/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 338 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/338/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 352 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/352/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 353 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/353/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 361 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/361/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 362 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/362/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 363 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/363/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 374 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/374/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 375 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/375/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 377 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/377/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 383 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/383/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 385 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/385/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 387 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/387/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 388 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/388/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 391 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/391/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 392 (sh) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/392/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Inspecting process environment variables. PID: 403 (npm ci) [WORKER 2023-10-03 04:18:31Z INFO LinuxProcessExtensions] Read env from /proc/403/environ [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Wait for all connectivity checks to finish. [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Connectivity check result: https://acghubeus0.actions.githubusercontent.com/_apis/health: OK [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Connectivity check result: https://actions-results-receiver-production.githubapp.com/health: OK [WORKER 2023-10-03 04:18:31Z INFO JobExtension] Connectivity check result: https://pipelinesghubeus13.actions.githubusercontent.com/_apis/health: OK [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext] Publish step telemetry for current step { [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext]   "action": "complete_job", [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext]   "type": "runner", [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext]   "stage": "Post", [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext]   "stepId": "47ad5bda-ffab-4f20-9c50-5801efd60e32", [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext]   "result": "succeeded", [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext]   "errorMessages": [], [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext]   "executionTimeInSeconds": 1, [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext]   "startTime": "2023-10-03T04:18:30.9698566Z", [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext]   "finishTime": "2023-10-03T04:18:31.8481395Z" [WORKER 2023-10-03 04:18:31Z INFO ExecutionContext] }. [WORKER 2023-10-03 04:18:31Z INFO JobRunner] Job result after all job steps finish: Failed [WORKER 2023-10-03 04:18:31Z INFO JobRunner] Completing the job execution context. [WORKER 2023-10-03 04:18:31Z INFO RunnerServer] EstablishVssConnection [WORKER 2023-10-03 04:18:31Z INFO RunnerServer] Establish connection with 100 seconds timeout. [WORKER 2023-10-03 04:18:31Z INFO JobServerQueue] Try to append 1 batches web console lines for record '47ad5bda-ffab-4f20-9c50-5801efd60e32', success rate: 1/1. [WORKER 2023-10-03 04:18:31Z INFO JobServerQueue] Try to append 1 batches web console lines for record '71f433e8-a4c8-5292-e2f1-3c2eea32e952', success rate: 1/1. [WORKER 2023-10-03 04:18:31Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:31Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_8ddd3fe5-3d28-5aee-259c-90574a267e3a.1 [WORKER 2023-10-03 04:18:32Z INFO GitHubActionsService] Starting operation Location.GetConnectionData [WORKER 2023-10-03 04:18:32Z INFO RunnerServer] EstablishVssConnection [WORKER 2023-10-03 04:18:32Z INFO RunnerServer] Establish connection with 60 seconds timeout. [WORKER 2023-10-03 04:18:32Z INFO GitHubActionsService] Starting operation Location.GetConnectionData [WORKER 2023-10-03 04:18:32Z INFO RunnerServer] EstablishVssConnection [WORKER 2023-10-03 04:18:32Z INFO RunnerServer] Establish connection with 60 seconds timeout. [WORKER 2023-10-03 04:18:32Z INFO GitHubActionsService] Starting operation Location.GetConnectionData [WORKER 2023-10-03 04:18:32Z INFO GitHubActionsService] Finished operation Location.GetConnectionData [WORKER 2023-10-03 04:18:32Z INFO GitHubActionsService] Finished operation Location.GetConnectionData [WORKER 2023-10-03 04:18:32Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:32Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_54313521-34e4-5b6e-7260-413bbe0cc494.1 [WORKER 2023-10-03 04:18:32Z INFO GitHubActionsService] Finished operation Location.GetConnectionData [WORKER 2023-10-03 04:18:32Z INFO JobRunner] Newer packages [ [WORKER 2023-10-03 04:18:32Z INFO JobRunner]   "2.309.0", [WORKER 2023-10-03 04:18:32Z INFO JobRunner]   "2.308.0", [WORKER 2023-10-03 04:18:32Z INFO JobRunner]   "2.307.1", [WORKER 2023-10-03 04:18:32Z INFO JobRunner]   "2.306.0", [WORKER 2023-10-03 04:18:32Z INFO JobRunner]   "2.305.0" [WORKER 2023-10-03 04:18:32Z INFO JobRunner] ] [WORKER 2023-10-03 04:18:32Z INFO JobRunner] Shutting down the job server queue. [WORKER 2023-10-03 04:18:32Z INFO JobServerQueue] Fire signal to shutdown all queues. [WORKER 2023-10-03 04:18:32Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:32Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_5b332ad0-3006-57c0-fee1-42721e212b7f.1 [WORKER 2023-10-03 04:18:33Z INFO JobServerQueue] Try to upload 11 log files or attachments, success rate: 11/11. [WORKER 2023-10-03 04:18:33Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:33Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_7b842c8c-e81b-5682-69d2-354eb0c5a8f4.1 [WORKER 2023-10-03 04:18:33Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:33Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_f3aa20a5-40de-5782-76d6-80330e8891c4.1 [WORKER 2023-10-03 04:18:34Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:34Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_26af32f2-7a4d-57a0-f38e-2a3d4c4f828d.1 [WORKER 2023-10-03 04:18:34Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:34Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_d15a3825-84ab-50ae-6588-81420ba0c568.1 [WORKER 2023-10-03 04:18:35Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:35Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_3d1c7a6b-8d7f-5623-4363-3827fb10d229.1 [WORKER 2023-10-03 04:18:35Z INFO JobServerQueue] Got a step log file to send to results service. [WORKER 2023-10-03 04:18:35Z INFO JobServerQueue] Starting upload of step log file to results service ResultsLog, /home/runner/_diag/blocks/1701621c-354f-430e-a272-5d1fed596531_ed69047a-21d1-5c3b-2c14-ef3846531d25.1 [WORKER 2023-10-03 04:18:36Z INFO JobServerQueue] Tried to upload 11 file(s) to results, success rate: 11/11. [WORKER 2023-10-03 04:18:37Z INFO JobServerQueue] All queue process task stopped. [WORKER 2023-10-03 04:18:37Z INFO JobServerQueue] Web console line queue drained. [WORKER 2023-10-03 04:18:37Z INFO JobServerQueue] Uploading 8 files in one shot. rpc error: code = NotFound desc = an error occurred when try to find container "eac843a89726083fe6f43296f7a4cafc9321dc118d045a5a33ad12e376f3f0be": not found% `

Github UI logs:
`##[debug]Evaluating` condition for step: 'Post Check out repository code' ##[debug]Evaluating: always() ##[debug]Evaluating always: ##[debug]=> true ##[debug]Result: true ##[debug]Starting: Post Check out repository code ##[debug]Re-evaluate condition on job cancellation for step: 'Post Check out repository code'. ##[debug]Skip Re-evaluate condition on runner shutdown. ##[debug]Loading inputs ##[debug]Evaluating: github.repository ##[debug]Evaluating Index: ##[debug]..Evaluating github: ##[debug]..=> Object ##[debug]..Evaluating String: ##[debug]..=> 'repository' ##[debug]=> 'Simpplr/Simpplr-Notification' ##[debug]Result: 'Simpplr/Simpplr-Notification' ##[debug]Evaluating: github.token ##[debug]Evaluating Index: ##[debug]..Evaluating github: ##[debug]..=> Object ##[debug]..Evaluating String: ##[debug]..=> 'token' ##[debug]=> '***' ##[debug]Result: '***' ##[debug]Loading env Post job cleanup. Error: The operation was canceled. ##[debug]System.OperationCanceledException: The operation was canceled. ##[debug]   at System.Threading.CancellationToken.ThrowOperationCanceledException() ##[debug]   at GitHub.Runner.Sdk.ProcessInvoker.ExecuteAsync(String workingDirectory, String fileName, String arguments, IDictionary2 environment, Boolean requireExitCodeZero, Encoding outputEncoding, Boolean killProcessOnCancel, Channel1 redirectStandardIn, Boolean inheritConsoleHandler, Boolean keepStandardInOpen, Boolean highPriorityProcess, CancellationToken cancellationToken) ##[debug]   at GitHub.Runner.Common.ProcessInvokerWrapper.ExecuteAsync(String workingDirectory, String fileName, String arguments, IDictionary2 environment, Boolean requireExitCodeZero, Encoding outputEncoding, Boolean killProcessOnCancel, Channel1 redirectStandardIn, Boolean inheritConsoleHandler, Boolean keepStandardInOpen, Boolean highPriorityProcess, CancellationToken cancellationToken) ##[debug]   at GitHub.Runner.Worker.Handlers.DefaultStepHost.ExecuteAsync(IExecutionContext context, String workingDirectory, String fileName, String arguments, IDictionary2 environment, Boolean requireExitCodeZero, Encoding outputEncoding, Boolean killProcessOnCancel, Boolean inheritConsoleHandler, String standardInInput, CancellationToken cancellationToken)
##[debug] at GitHub.Runner.Worker.Handlers.NodeScriptActionHandler.RunAsync(ActionRunStage stage)
##[debug] at GitHub.Runner.Worker.ActionRunner.RunAsync()
##[debug] at GitHub.Runner.Worker.StepsRunner.RunStepAsync(IStep step, CancellationToken jobCancellationToken)
##[debug]Finishing: Post Check out repository code`

**Comment by tarasmadan** [comment]
Exit code 130 was my problem during the experiments with [spot VMs](https://cloud.google.com/spot-vms).
It looks like the VM with job is moved out from the cluster which generates this error.
